assets/scss/modular/_resource-filter.scss

```diff
@@ -0,0 +1,42 @@
+
+/*
+
+---
+name: Filter
+category:
+ - Components/Resources
+---
+
+```html
+<label class="resource-filter / custom-checkbox">
+	<input type="checkbox" value="annual-report">
+	<span><svg><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-close"></use></svg></span>
+	Annual Report
+</label>
+```
+
+*/
+
+.resource-filter {
+
+	@include font('secondary');
+	@include font-size(17);
+	color: color('background');
+	background-color: color('white');
+	margin-bottom: em(10);
+	user-select: none;
+
+	@include mq(M, 'max-width') {
+		margin-bottom: spacing(3);
```

Comment:
Why not just set this as your base margin bottom, and then put the `margin-bottom: em(10);`  in your next media query?